modules/veteran/app/workers/veteran/base_reloader.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+
+module Veteran
+  class BaseReloader
+    include Sidekiq::Worker
+    BASE_URL = 'https://www.va.gov/ogc/apps/accreditation/'
+
+    private
+
+    def find_or_initialize(hash_object)
+      rep = Veteran::Service::Representative.find_or_initialize_by(representative_id: hash_object['Registration Num'],
+                                                                   first_name: hash_object['First Name'],
+                                                                   last_name: hash_object['Last Name'])
+      poa_code = hash_object['POA Code'].gsub!(/\W/, '')
+      rep.poa_codes << poa_code unless rep.user_types.include?(poa_code)
+      rep.phone = hash_object['Phone']
+      rep
+    end
+
+    def fetch_data(action)
+      page = Faraday.new(url: BASE_URL).post(action, id: 'frmExcelList', name: 'frmExcelList').body
+      doc = Nokogiri::HTML(page)
+      headers = doc.xpath('//table/tr').first.children.map { |child| child.children.text.scrub }
+      doc.xpath('//table/tr').map do |line|
+        row = line.children.map { |child| child.children.text.scrub }
```

Comment:
during your refactoring you were able to drop .scrub without causing a test to fail... I saw the comment that we need this, I'd recommend a failing test so that it can't be dropped in the future

Reply:
Will add it to another PR then just to close this one out for now since it's been open for longer than desired and that test may take longer to figure out because of the nature of the test payload, in essence the scrub was first removed because the data set I was using didn't special characters in the OGC data that can some times be there and sometimes not it just depends on when the data was pulled. Defaulting to scrubbing the data makes the most sense by default at the moment.